### ISSUE TITLE
paper-fab accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-fab#6]
There are some issues with navigation. 

Chrome / Chrome OS / ChromeVox: Unable to navigate to the buttons using the keyboard / spoken feedback, but this is a limitation on the ChromeVox side. 

Chrome / Mac / VoiceOver: the navigation works fine, but the buttons are announced as "group", not button. 

Chrome / Windows / NVDA: NVDA does read out the buttons, but it speaks them two at a time, not individually. When using the Tab key, the navigation works fine and I receive proper spoken feedback. 

Firefox / Windows / NVDA: The buttons' labels are not read. They are simply read as "clickable". In between the top two buttons, NVDA is also reading out "blank", as if it thinks there is a blank item in between them. 

Focus / contrast: 

Focus is not visible. 
The contrast of the yellow and white icon is not sufficient. It's nearly impossible to see what that icon is.

**Comment by cdata** [comment]
This should be substantially improved as of https://github.com/PolymerElements/paper-behaviors/pull/15

**Comment by notwaldorf** [comment]
I think this should be fixed. @lpalmaro, would you mind double checking, please?
